### PR TITLE
physic: Adds flag.Value for RelativeHumidity

### PR DESCRIPTION
**Pull request by NeuralSpaz**
This adds support for the flag.Value interface for RelativeHumidity unit type.
Add Set and flag doc examples.
Add Set() for RelativeHumidity.
Add Tests for RelativeHumidity Set().
Add Set() and flag.Value examples for RelativeHumidity.

Helps with #303.

### CODE REVIEW

File: conn/physic/example_test.go [hunk 1]
```diff
@@ -393,6 +393,30 @@ func ExampleRelativeHumidity() {
...
 
+func ExampleRelativeHumidity_Set() {
+	var r physic.RelativeHumidity
+
+	if err := r.Set("50.6%rH"); err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println(r)
+
+	if err := r.Set("20%rH"); err != nil {
```
Comment (maruel) [inline conn/physic/example_test.go]: You can put "20%" in the example, to show that it works?

File: conn/physic/units.go [hunk 2]
```diff
@@ -920,11 +920,60 @@ func (h RelativeHumidity) String() string {
...
+	return nil
+}
+
 const (
 	TenthMicroRH RelativeHumidity = 1                 // 0.00001%rH
 	MicroRH      RelativeHumidity = 10 * TenthMicroRH // 0.0001%rH
 	MilliRH      RelativeHumidity = 1000 * MicroRH    // 0.1%rH
 	PercentRH    RelativeHumidity = 10 * MilliRH      // 1%rH
+
+	maxRelativeHumidity RelativeHumidity = (1 << 31) - 1
```
Comment (maruel) [inline conn/physic/units.go]: It's between 0% and 100%.
https://periph.io/x/periph/conn/physic#RelativeHumidity

I prefer to deny parsing values outside this range, or at the very least values below 0.

File: conn/physic/example_test.go (hunk 1 above)
Comment (NeuralSpaz) [reply conn/physic/example_test.go]: sure can.

**Comment by maruel** [comment]
Humm looks like the beaglebone died again.

**Pull request merged**